packages/core/src/generators/interface.ts

```diff
@@ -46,7 +46,10 @@ export const generateInterface = ({
     scalar.type === 'object' &&
     !context?.output.override?.useTypeOverInterfaces
   ) {
-    model += `export interface ${name} ${scalar.value}\n`;
+    // If `scalar.value` is 'unknown', replace it with `{}` to avoid type error
+    const blankIntefaceValue = scalar.value === 'unknown' ? '{}' : scalar.value;
```

Comment:
```suggestion
    const blankInterfaceValue = scalar.value === 'unknown' ? '{}' : scalar.value;
```

Reply:
I fixed it 

https://github.com/anymaniax/orval/pull/1228/commits/564dd8ee2015b91ce23089c7d11fd3cc9ec88b1f